src/Modules/OrchardCore.Commerce/Views/StripeCheckout.cshtml

```diff
@@ -33,9 +33,7 @@
     </button>
```

Comment:
It's because the card change event cleared the top (generic) error box, but not the other way around. I've fixed this now, displaying any error clears either boxes first.

The bottom box is for live notifications, e.g. when it says "Your card number is invalid." during typing. I had to make two boxes because you wouldn't want it to scroll all the way up while you are trying to type in the card details. Sigh, that's even more complexity to untangle in #149 ...